src/test/java/org/kohsuke/github/UserTest.java

```diff
@@ -27,4 +26,27 @@ public void listFollowsAndFollowers() throws IOException {
         assertEquals(30, users.size());
         return users;
     }
+
+    @Test
+    public void getKeys() throws IOException {
```

Comment:
This is really great.  How was it using this test framework and `takeSnapshot`?  I just added it and would appreciate any feedback you can offer.

Reply:
It took a while before I understood the framework but then it was simple. Deleting and generating the test data has drawbacks since it is easy  do forget about that step but the advantage with test taking lots of time and that data is constant is a greater benefit.

Reply:
Okay, well, if you have any suggestions for how to make it easier (such as improvements to the CONTRIBUTING.md) that would be great.